frontend/src/features/contacts/contact/detail/Organization.tsx

```diff
@@ -0,0 +1,215 @@
+import { FormSection } from 'components/common/form/styles';
+import { ContactMethodTypes } from 'constants/contactMethodType';
+import { Dictionary } from 'interfaces/Dictionary';
+import {
+  IContactAddress,
+  IContactMethod,
+  IContactOrganization,
+  IContactPerson,
+} from 'interfaces/IContact';
+import * as React from 'react';
+import { Col } from 'react-bootstrap';
+import { FaCircle, FaRegBuilding } from 'react-icons/fa';
+import { Link } from 'react-router-dom';
+import { phoneFormatter } from 'utils/formUtils';
+
+import * as Styled from '../../styles';
+
+export interface OrganizationViewProps {
+  organization: IContactOrganization;
+}
+
+interface ContactInfoField {
+  info: string;
+  label: string;
+}
+
+interface AddressField {
+  address: IContactAddress;
+  label: string;
+}
+
+const OrganizationView: React.FunctionComponent<OrganizationViewProps> = ({ organization }) => {
+  const phoneTypes: Dictionary<string> = {};
+  phoneTypes[ContactMethodTypes.WorkMobil] = 'Mobile';
+  phoneTypes[ContactMethodTypes.WorkPhone] = 'Work';
+  phoneTypes[ContactMethodTypes.PersPhone] = 'Home';
+  phoneTypes[ContactMethodTypes.Fax] = 'Fax';
+
+  const personPhoneNumbers: ContactInfoField[] = getContactInfo(organization, phoneTypes);
+
+  const emailTypes: Dictionary<string> = {};
+  emailTypes[ContactMethodTypes.WorkEmail] = 'Work';
+  emailTypes[ContactMethodTypes.PerseEmail] = 'Personal';
+  const personEmails: ContactInfoField[] = getContactInfo(organization, emailTypes);
+
+  let personAddresses: AddressField[];
+  if (organization.addresses === undefined) {
+    personAddresses = [];
+  } else {
+    personAddresses = organization.addresses.reduce(
+      (accumulator: AddressField[], value: IContactAddress) => {
+        accumulator.push({ label: value.addressType.description, address: value });
+
+        return accumulator;
+      },
+      [],
+    );
+  }
+
+  return (
+    <>
+      <FormSection key={'contact-org-' + organization.id + '-names'} className="mb-4">
+        <Styled.RowAligned>
+          <Col>
+            <Styled.H2>
+              <FaRegBuilding size={20} className="mr-2" />
+              <span data-testid="contact-organization-fullname">{organization.name}</span>
+            </Styled.H2>
+          </Col>
+          <Col md="auto" className="ml-auto">
+            <Styled.StatusIndicators className={organization.isDisabled ? 'inactive' : 'active'}>
+              <FaCircle size={7} className="mr-2" />
```

Comment:
Nit: think the icon bobbi is using is different, but I wouldn't want you to waste time on this either.

Reply:
Uhm, dont know which one Bobi is using

Reply:
you can see it by clicking on it in uxpin (a download option appears). alternatively she will sometimes document in the uxpin comments section.